cartography/intel/aws/ec2.py

```diff
@@ -737,6 +737,7 @@ def load_ec2_vpcs(session, data, current_aws_account_id, aws_update_tag):
     new_vpc.is_default = {IsDefault},
     new_vpc.primary_cidr_block = {PrimaryCIDRBlock},
     new_vpc.dhcp_options_id = {DhcpOptionsId},
+    new_vpc.region = {Region},
```

Comment:
Non-blocking question: `requestor_vpc` doesn't have a `region` (probably because these can be "foreign".  Are we able to infer the region of these somehow? 

Just wanted to point out that some VPC nodes may have the `region` field defined and others might not, so it might be helpful to point this out in our docs.

Reply:
correct. only foreign will have no region